src/lib/Notifications.svelte

```diff
@@ -216,6 +227,8 @@
     const persist = options.persist
     const computedTimeout = options.persist ? 0 : (options.timeout || timeout)
     const id = Math.random().toString(36).replace(/[^a-z]+/g, '')
+    const showProgress = options.showProgress ?? true // Default showProgress to true
+    const icon = options.icon // Should be a svelte component
```

Comment:
```suggestion
    const icon = options.icon
```

I'm anti-comments :)